twitterscraper/query.py

```diff
@@ -87,8 +87,8 @@ def query_tweets_once(query, limit=None):
     try:
         while True:
             new_tweets, pos = query_single_page(
-                INIT_URL.format(q=query) if pos is None
-                else RELOAD_URL.format(q=query, pos=pos),
+                INIT_URL.format(q=query, lang=lang) if pos is None
```

Comment:
architecture idea, take `**kwargs` and pass them like that to format, that'll always allow the user to pass everything that you allow in the URL format string.